book/chapters/chapter10/advanced_technical_aspects_of_mlr3.qmd

```diff
@@ -241,58 +241,133 @@ See @sec-hpc-exec for larger benchmark experiments that may have a cumulative ru
 
 ### Parallelization of Tuning {#sec-parallel-tuning}
 
-The parallelization of tuning is a special case of benchmarking.
-The `r ref("Tuner")` proposes a batch of `batch_size` hyperparameter configurations and creates `batch_size` learners (one learner parameterized for each proposed configuration).
-All learners are then passed to `r ref("benchmark()")` together with the resampling strategy of the tuning instance.
-
-For many optimizers, you can adjust the `batch_size`, i.e., control how many hyperparameter configurations are evaluated in parallel during a call of `benchmark()`.
-If you want full parallelization, make sure that the batch size multiplied by the number of resampling iterations is at least equal to the number of available workers.
-If you expect homogeneous runtimes, i.e., you are tuning over a single learner or pipeline, and you have no hyperparameter with a large influence on the runtime, aim for a multiple of the number of workers.
+Tuning is usually an iterative procedure, consisting of steps that are themselves embarrassingly parallel.
+In each iteration, a tuner proposes a batch of hyperparameter configurations -- in the worst case of size 1 -- which can then be evaluated in parallel.
+After each iteration, a typical tuner adapts itself in some way based on the obtained performance values.
+Random and grid search are exceptions, since they do not choose configurations based on past results.
+For them, all evaluations are therefore independent and can in principle be fully parallelized.
+Otherwise, tuning can be regarded as iterative benchmarks, which is also how `mlr3` implements it:
+The `r ref("Tuner")` proposes a batch of learners, each with a different configuration in its `$param_set$values`.
+The size of this batch can often be controlled with a tuner's `batch_size` configuration parameter.
+This batch is then passed to `r ref("benchmark()")` together with the resampling strategy of the tuning instance.
+
+Since each `benchmark()`-call depends on previous results, it is generally not possible to parallelize tuning at a higher "level" than individual benchmarks.
```

Comment:
```suggestion
Since each call to `benchmark()` depends on previous results, it is generally not possible to parallelize tuning at a higher "level" than individual benchmarks.
```